### DISCUSSION TITLE
Exclude 1 Device from availability check

### DISCUSSION THREAD

**Discussion by jonnycastaway** [Koenkk/zigbee2mqtt#24932] [General]
Hi @ all.
I have > 100 Devices in my z2m system and check the availability of them with "availability advanced" activated.
Now i have 1 Device (Hue Tap Switch) that only reports once it was used. After 25h it's falls in "available: false" state. That's ok because it's powered by a piezo mechanism (no batterys). But i generate a Pushover Message when a Device is going to "available: false" state and now would exclude the Tap Switch from the check.
In the Documentation i found the point where is described how to activate each device independently. But can i also deactivate one device without specifying a list of all devices (100 enabled and 1 disabled)?
I didn't want to maintain such a list to get 1 device disabled.

**Comment by chris-1243** [comment]
Do it like this in the device list

```
'0x000000000....':
  friendly_name: xxx
  description: yyy
  availability: false
```

https://www.zigbee2mqtt.io/guide/configuration/device-availability.html#availability-advanced-configuration

**Reply by jonnycastaway** [reply]
Argh, im so stupid. I missunderstood the part with "don't add availability: true".
Sorry for my stupidity.
